reana-ui/src/pages/gitlabProjects/GitLabProjects.js

```diff
@@ -91,7 +104,7 @@ export default function GitLabProjects() {
     return (
       <div>
         <TopHeader />
-        <Container text className="gitlab-container">
+        <Container text className="container gitlab-container">
```

Comment:
`gitlab-container` class is deleted, right? (note also `no-projects-container` at the end of the file)

Reply:
Indeed, it's a leftover from previous changes, thanks! 👍